Makefile

```diff
@@ -69,13 +69,13 @@ endif
 	glide install -v
 
 	echo "Building dependencies to make builds faster"
-	go install github.com/heketi/heketi
+	#go install github.com/heketi/heketi
```

Comment:
1. Why comment out and not remove?
2. AFAICT, this is not about making the build fast, but about removing something that you think is harmful. Could you please create a second patch for it?